index.html

```diff
@@ -302,12 +302,25 @@ <h2 class="about-title">About<br />Myself<br /></h2>
       <hr />
       <div class="skills-container">
         <a class="skills--title-link" href="#">
-          <span class="skills--title">Skills</span></a
-        >
+          <span class="skills--title">Skills</span></a>
         <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
       </div>
+      <hr class="underline">
     </section>
 
+    <section class="contact-form-sec">
+    <h1 class="contact-me">Contact me</h1>
+    <p class="contact-paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```

Comment:
I suggest you use any of the(h1-h2) elements for your text. A good job is done congratulations.

Reply:
Good work so far.